requirements.txt

```diff
@@ -1 +1 @@
-opencv-contrib-python~=4.6.0.66  # for usage with GStreamer, one has to compile OpenCV with its support turned on
\ No newline at end of file
+opencv-python~=4.6.0.37  # for usage with GStreamer, one has to compile OpenCV with its support turned on
```

Comment:
did you check it could be installed? because PyPI does not have this version listed...

Reply:
I just tested it, in case the opencv is already installed via whl file, this variant does not reinstall it, in case the opencv is not installed, it will install 4.6.0.66 (the only released 4.6.0 opencv on PyPI).